packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header-item.component.tsx

```diff
@@ -25,16 +27,18 @@ const VitalsHeaderItem: React.FC<VitalsHeaderItemProps> = ({ interpretation, val
         })}
       >
         <div className={styles['label-container']}>
-          <label className={styles.label}>{unitName}</label>
+          <label htmlFor={valueContainerId} className={styles.label}>
+            {unitName}
+          </label>
           {flaggedAbnormal ? (
             <span className={styles[interpretation.replace('_', '-')]} title={t('abnormalValue', 'Abnormal value')} />
           ) : null}
         </div>
         <div className={styles['value-container']}>
-          <label className={styles['pad-right']}>
-            <span className={styles.value}>{value || '-'} </span>
-            <span className={styles.units}>{value && unitSymbol}</span>
-          </label>
+          <div id={valueContainerId} className={styles['pad-right']}>
+            <span className={styles.value}>{value || '-'}</span>
+            <span className={styles.units}> {value && unitSymbol}</span>
+          </div>
```

Comment:
![image](https://github.com/user-attachments/assets/51290e63-b5dc-4b60-94c3-a7ec034138bf)

If you are implementing with accessibility in mind, Example 1 should be implemented.

The mapping with `htmlFor` is only used for form elements.
![image](https://github.com/user-attachments/assets/f341df19-1704-40bd-a6b2-88dd7cce38fb)